README.rst

```diff
@@ -4,13 +4,9 @@
 efficiently evaluate mathematical expressions involving multi-dimensional
 arrays.  It can use GPUs and perform efficient symbolic differentiation.
 
-This is a fork of the `original Theano library <https://github.com/Theano/Theano>`__ that is being
+This is a fork of the `original Theano library <https://github.com/pymc-devs/aesara>`__ that is being
```

Comment:
This link should still point to original theano repo

Reply:
Yes, it should.